### PR TITLE
Add tkittel as maintainer

### PR DESCRIPTION
**Pull request by tkittel**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ source:
   folder: dataset
 
 build:
-  number: 0
+  number: 1
```
Comment (tschoonj) [inline recipe/meta.yaml:19]: No need to bump here.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -43,3 +43,4 @@ extra:
   recipe-maintainers:
     - tschoonj
     - chrisburr
+    - tkittel
```
Comment (tschoonj) [inline recipe/meta.yaml:45]: Can you remove me from the list please?

**Comment by tkittel** [comment]
@tschoonj OK I will remove you from the maintainer list in all the PRs.

For the build number, I would like to increment it since I am seeing some weird issues and I think it would be nice with freshly rerendered and rebuilt packages.

**Comment by tkittel** [comment]
@conda-forge-admin, please rerender

**Comment by tkittel** [comment]
For all the PRs:

@tschoonj it seems a lot of the tests are failing, but since I am not yet a maintainer I can't really investigate.

Do you want me to back out the build number increments, and only do the maintainer change in the PRs? Then I guess they are safe to merge even with failing tests?

**Comment by tkittel** [comment]
Or actually, now I was able to see a log file, so maybe I can actually investigate.

**Comment by tkittel** [comment]
Ah, it seems there are sha mismatches for some reason. @drbenmorgan do you know if geant4 is in the habit of updating the online dataset tarballs in-place occasionally?

In any case @tschoonj how do you want to proceed? Should I try to fix all the sha's in all the PRs?

**Comment by tkittel** [comment]
@tschoonj so I fixed the sha values in the geant4-data-xxx-feedstocks where it was apparently needed, so all of those PRs are now passing. The checks in the geant-feedstock of course takes longer to finish.

**Pull request merged**